jsx/Form.js

```diff
@@ -438,7 +438,9 @@ class SelectElement extends Component {
     }
 
     // Add empty option
-    if (this.props.emptyOption) {
+    // If only one option exists, don't add empty option so that the only available
+    // option is selected by default
+    if (this.props.emptyOption && Object.keys(this.props.options).length != 1) {
```

Comment:
I think this only makes sense if it's a required element. If it's not required, both empty and the 1 option are valid values.

Reply:
see latest commit